lib/fluent/plugin/in_tail.rb

```diff
@@ -103,7 +105,7 @@ def configure(conf)
     end
 
     def configure_parser(conf)
-      @parser = Plugin.new_parser(conf['format'])
+      @parser = Fluent::Plugin.new_parser(conf['format'])
```

Comment:
Use `parser` plugin helper to instantiate it with owner's attributes (e.g., plugin id and logger)